File: sudoq-app/sudoqmodel/src/test/java/de/sudoq/model/solverGenerator/solver/ComplexityRelationTests.java
package de.sudoq.model.solverGenerator.solver;


import static org.junit.jupiter.api.Assertions.assertEquals;


import org.junit.jupiter.api.Test;

import de.sudoq.model.solverGenerator.solver.ComplexityRelation;

class ComplexityRelationTests {

    @Test
    void test() {
		ComplexityRelation[] types = ComplexityRelation.values();
		for (ComplexityRelation type : types) {
            assertEquals(ComplexityRelation.valueOf(type.toString()), type);
		}
	}

}

File: sudoq-app/sudoqmodel/src/test/kotlin/de/sudoq/model/solverGenerator/solver/ComplexityRelationTests.kt
package de.sudoq.model.solverGenerator.solver

import org.amshove.kluent.`should be equal to`
import org.junit.jupiter.api.Test


internal class ComplexityRelationTests {
    @Test
    fun test() {
        val types = ComplexityRelation.entries.toTypedArray()
        types.forEach { it `should be equal to` ComplexityRelation.valueOf(it.toString()) }
    }
}
